governance/xc_admin/packages/xc_admin_cli/README.md

```diff
@@ -0,0 +1,28 @@
+# Pyth Cross-chain Governance Admin CLI Program
+
+This program allows you to do a couple of features such as:
```

Comment:
I think this should probably be a high level description of what the program does

Reply:
Which is basically "interacting with the multisig"